Src/PerformanceCollector/Shared/Implementation/PerformanceCounterUtility.cs

```diff
@@ -272,6 +272,12 @@ internal static IList<string> GetClrProcessInstances()
             }
 
             usesPlaceholder = true;
+
+            if (IsWebAppRunningInAzure())
+            {
+                return instanceName;
```

Comment:
styling: please try to avoid returning from the middle of the method. I think you need to check this condition first before any other conditions.